### PR TITLE
Added technique for analyzing entitlements

### PR DESCRIPTION
**Pull request by TheDauntless**
This PR does the following:

* Add MASTG-TECH-0111: Extracting entitlements from a MachO binary
* Add tool page for ldid 
* Updated references to previous tools to now refer to technique
* Fixed a few markdown issues in related pages.

### CODE REVIEW

File: techniques/ios/MASTG-TECH-0084.md [hunk 1]
```diff
@@ -55,55 +55,55 @@ Note: On iOS 12 and higher, use the following procedure to sign the debugserver
 
 2) Connect to the device via SSH and create the file, named entitlements.xml, with the following content:
 
-    ```xml
```
Comment (TheDauntless) [inline techniques/ios/MASTG-TECH-0084.md:58]: See https://github.com/DavidAnson/markdownlint/issues/125

**Comment by cpholguera** [comment]
would this be closing also https://github.com/OWASP/owasp-mastg/issues/2880?

File: techniques/ios/MASTG-TECH-0056.md
Comment (cpholguera) [inline techniques/ios/MASTG-TECH-0056.md:1]: The URL checker complains about this link in this file (not added by you):

ERROR: 1 dead links found!
[✖] http://repo.hackyouriphone.org/appsyncunified → Status: 0

File: techniques/ios/MASTG-TECH-0084.md (hunk 1 above)
Comment (cpholguera) [reply techniques/ios/MASTG-TECH-0084.md:58]: Was this reporting MD046: https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md046---code-block-style

**Review by cpholguera — APPROVED** [review]

File: techniques/ios/MASTG-TECH-0111.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+platform: ios
+---
+
+To extract the entitlements from a MachO binary, the following tools can be used:
+
+- @MASTG-TOOL-0111
+- @MASTG-TOOL-0105
+- @MASTG-TOOL-0101
+
+The different tools are used on the main binary of @MASTG-APP-0028, which contains two architectures.
```
Comment (cpholguera) [inline techniques/ios/MASTG-TECH-0111.md:12]:
```suggestion
The following examples use these tools on the main binary of @MASTG-APP-0028, which contains two architectures.
```

**Review by cpholguera — APPROVED** [review]

**Pull request merged**